frontend/public/static/locale/fr/translate.ftl

```diff
@@ -206,6 +206,9 @@ notification--entity-not-found = Impossible de charger la chaîne spécifiée
 otherlocales-Translation--copy =
     .title = Copier la traduction
 
+otherlocales-Translation--header-link = 
+    .title = Chaîne ouvre en { $locale } ({ $code })
```

Comment:
Sorry, that's not correct French (I'm guessing Google Translate? :smile: ). Please remove this instead, as we'll want to let Pontoon users do the translations themselves in the future.

Reply:
Yes sure :)